loom/engine/bindings/loom/lmGameController.h

```diff
@@ -0,0 +1,35 @@
+#include <SDL.h>
+
+#include "loom/script/loomscript.h"
+
+#define MAX_CONTROLLERS 4
+
+class LoomGameController
```

Comment:
What is the purpose of this class? What do the methods do? Are docs here in this file or elsewhere?